### DISCUSSION TITLE
EAP-TLS Local identity is not verified in recent version

### DISCUSSION THREAD

**Discussion by jkoendev** [strongswan/strongswan#872] [Q&A] [answered]
Hi

It seems that in EAP-TLS client certificates must have a SAN which is equal to the local identity (as in #646). Otherwise resulting in 

```
no trusted certificate found for 'xxx' to verify TLS peer
```

After upgrading from StrongSwan 5.9.1 to 5.9.4 the SAN field seems not to be checked anymore, and i am able to connect with a non matching SAN/missing SAN/non matching CN. Unfortunatly i could not find anything in Changelog.

Did this behavior indeed change recently? Is there a config to influence this behavior?

I am connecting from an iOS client.

Thanks

**Comment by tobiasbrunner** [comment, marked as answer]
Please see #873.
